programmers/제일-작은-수-제거하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(arr) {
+   const min = Math.min(...arr);
+   const minIndex =  arr.indexOf(min);
+   
+   arr.splice(minIndex, 1);
+   if (arr.length < 1) arr.push(-1);
+   return arr;
+}
```

Comment:
min은 MIN_NUM 변수명이 더 나아보임 잘풀었음
```suggestion
function solution(arr) {
   const MIN_NUM = Math.min(...arr);
   const minIndex = arr.indexOf(MIN_NUM);
   
   arr.splice(minIndex, 1);
   
   if (arr.length < 1) arr.push(-1);
   return arr;
}
```